packages/grafana-ui/src/components/Select/Select.tsx

```diff
@@ -12,7 +12,7 @@ export function MultiSelect<T>(props: MultiSelectCommonProps<T>) {
   return <SelectBase {...props} isMulti />;
 }
 
-interface AsyncSelectProps<T> extends Omit<SelectCommonProps<T>, 'options'>, SelectAsyncProps<T> {
+export interface AsyncSelectProps<T> extends Omit<SelectCommonProps<T>, 'options'>, SelectAsyncProps<T> {
```

Comment:
Is it a good idea to mess with grafana packages? This might change if the package is updated or if we need to merge the latest version. One alternative is to define this type ourselves.

Reply:
Yeah, good thought. Just added own typings 👍